hypothesis-python/tests/py3/test_asyncio.py

```diff
@@ -54,6 +57,7 @@ def g():
         if error is not None:
             raise error
 
+    @pytest.mark.skipif(PYPY, reason='Error in asyncio.new_event_loop()')
```

Comment:
nit: this doesn’t match the commit message, does it? This skips the test on PyPy3 *and* PyPy2.

Reply:
Yes, but it's a Python-3-only test!